tool/update-github-config.sh

```diff
@@ -0,0 +1,55 @@
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Why not dart?? :)

Reply:
Shelling out to `hub` and `git ls-files` is easier to bash... it's just a quick and dirty script, I wasn't even sure if it was worth checking in :)